packages/components/src/test/properties/UniquePropertyValuesSelector.test.tsx

```diff
@@ -165,6 +165,21 @@ describe("UniquePropertyValuesSelector", () => {
     expect(queryByText("unique-values-property-editor.select-values")).to.not.be.null;
   });
 
+  it("sets empty value text if provided value is an empty string", async () => {
+    const { container } = render(
+      <UniquePropertyValuesSelector
+        property={propertyDescription}
+        onChange={() => {}}
+        imodel={testImodel}
+        descriptor={descriptor}
+        value={{ valueFormat: PropertyValueFormat.Primitive, displayValue: '[""]', value: '[""]' }}
+      />,
+    );
+    await waitFor(() => expect(container.querySelector(".iui-tag-label")?.innerHTML).to.include("Empty Value"));
+  });
+
+  // This test does not represent a real situation, since one would only be able to create a tag-label if it is selected in the filter
```

Comment:
why do we need the test then?

Reply:
This test was written before, I decided to leave it since, I couldn't find a better way to reach a branch where an option has undefined display value in [this line](https://github.com/iTwin/presentation/pull/278/files/2b15c69cc6dde5c85e34bd7873d58fa533370c62#diff-cd3479e259eb8fe117da9a547677eb07e3a6fdef707a4de33927511785bea27cR85).